src/app/components/Modals/ConnectModal.tsx

```diff
@@ -224,7 +224,10 @@ export const ConnectModal: React.FC<ConnectModalProps> = ({
                           >
                             <AiOutlineInfoCircle />
                           </span>
-                          <Tooltip id={name} />
+                          <Tooltip
+                            id={name}
+                            className="max-w-xs whitespace-normal break-words"
```

Comment:
On all of them we added the class name `tooltip-wrap`, what's different about this tooltip?